lib/cretonne/src/abi.rs

```diff
@@ -81,13 +81,13 @@ impl ValueConversion {
 /// This will be implemented by individual ISAs.
 pub trait ArgAssigner {
     /// Pick an assignment action for function argument (or return value) `arg`.
-    fn assign(&mut self, arg: &AbiParam) -> ArgAction;
+    fn assign(&mut self, arg: &AbiParam, call_conv: CallConv) -> ArgAction;
```

Comment:
Instead of adding a `call_conv` parameter to this trait, you can add a `call_conv` member to the `struct Args` type defined in `intel/abi.rs`. This avoids threading this information through the ISA-independent code.